update docking demo to use nodeFlags

dcnieho opened the pull request:
This add the node flags. Most of the patch is in the attached below file since else it gets a little hard to send given thats on a different repo. You should be able to git apply that to hello_imgui

[0001-add-support-for-setting-node-flags-of-new-dock-space.patch](https://github.com/pthom/imgui_bundle/files/10930253/0001-add-support-for-setting-node-flags-of-new-dock-space.patch)

pthom commented:
Ok, I merged those changes and added the patch to hello_imgui. 
I had to change the generator options for DockNodeFlagsPrivate_

I also added a demo usage in python: 

https://github.com/pthom/imgui_bundle/blob/9dd5427bb6b65a2e6e9f631f6d72204cd3842297/bindings/imgui_bundle/demos_python/demos_immapp/demo_docking.py#L216-L217

Thanks!

dcnieho commented:
Hi Pascal,

Very nice, thanks!

Cheers,
Dee

On Thu, Mar 9, 2023 at 5:40 PM Pascal Thomet ***@***.***>
wrote:

> Ok, I merged those changes and added the patch to hello_imgui.
> I had to change the generator options for DockNodeFlagsPrivate_
>
> I also added a demo usage in python:
>
>
> https://github.com/pthom/imgui_bundle/blob/9dd5427bb6b65a2e6e9f631f6d72204cd3842297/bindings/imgui_bundle/demos_python/demos_immapp/demo_docking.py#L216-L217
>
> Thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pthom/imgui_bundle/pull/92#issuecomment-1462385123>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AANUOGND4GDOHIWFH56FGHLW3IBYVANCNFSM6AAAAAAVU45FYM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
